impl/src/main/java/org/glassfish/json/JsonParserImpl.java

```diff
@@ -55,25 +56,29 @@
 public class JsonParserImpl implements JsonParser {
 
     private final BufferPool bufferPool;
+    private final boolean allowDuplicateKeys;
     private Context currentContext = new NoneContext();
     private Event currentEvent;
 
     private final Stack stack = new Stack();
     private final JsonTokenizer tokenizer;
 
-    public JsonParserImpl(Reader reader, BufferPool bufferPool) {
```

Comment:
if this is reimplemented to sth like:
```
public JsonParserImpl(Reader reader, BufferPool bufferPool) {
    this(reader, bufferPool, true);
}
```
most changes done could have been avoided

Reply:
I made the change that you have recommended. Please see my latest commit.